### PR TITLE
removal of opening date field

### PR DESCRIPTION
**Pull request by danielryannimble**
removal of opening date field via migration

removal of openingdate field from domain and api layers

updating of tests

### CODE REVIEW

File: Dfe.Academies.Academisation.Service/Mappers/Legacy/ProjectAggregate/LegacyProjectDetailsMapper.cs [hunk 1]
```diff
@@ -25,7 +25,6 @@ internal static ProjectDetails MapNonEmptyFields(this LegacyProjectServiceModel
 			ApplicationReceivedDate = detailsToUpdate.ApplicationReceivedDate ?? existingProject.Details.ApplicationReceivedDate,
 			AssignedDate = detailsToUpdate.AssignedDate ?? existingProject.Details.AssignedDate,
 			HeadTeacherBoardDate = detailsToUpdate.HeadTeacherBoardDate.Equals(default(DateTime)) ? null : detailsToUpdate.HeadTeacherBoardDate ?? existingProject.Details.HeadTeacherBoardDate,
-			OpeningDate = detailsToUpdate.OpeningDate ?? existingProject.Details.OpeningDate,
```
Comment (dneed-nimble) [inline Dfe.Academies.Academisation.Service/Mappers/Legacy/ProjectAggregate/LegacyProjectDetailsMapper.cs:28]: You need to map the new field

File: Dfe.Academies.Academisation.Service/Mappers/Legacy/ProjectAggregate/LegacyProjectServiceModelMapper.cs [hunk 2]
```diff
@@ -20,7 +20,6 @@ internal static LegacyProjectServiceModel MapToServiceModel(this IProject projec
 			ApplicationReceivedDate = project.Details.ApplicationReceivedDate,
 			AssignedDate = project.Details.AssignedDate,
 			HeadTeacherBoardDate = project.Details.HeadTeacherBoardDate,
-			OpeningDate = project.Details.OpeningDate,
```
Comment (dneed-nimble) [inline Dfe.Academies.Academisation.Service/Mappers/Legacy/ProjectAggregate/LegacyProjectServiceModelMapper.cs:23]: Map new field

File: Dfe.Academies.Academisation.Service/Mappers/Legacy/ProjectAggregate/LegacyProjectDetailsMapper.cs (hunk 1 above)
Comment (danielryannimble) [reply Dfe.Academies.Academisation.Service/Mappers/Legacy/ProjectAggregate/LegacyProjectDetailsMapper.cs:28]: its already mapped?  line 54

File: Dfe.Academies.Academisation.Service/Mappers/Legacy/ProjectAggregate/LegacyProjectServiceModelMapper.cs (hunk 2 above)
Comment (danielryannimble) [reply Dfe.Academies.Academisation.Service/Mappers/Legacy/ProjectAggregate/LegacyProjectServiceModelMapper.cs:23]: is it already mapped line 49?

File: release-notes.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
 ## 9.0.1
+* Bug 132060 Added migration to remove OpeningDate field from projects
```
Comment (dneed-nimble) [inline release-notes.md:2]: Cheers for doing these Dan, could you follow the convention on the other repos/this repo?

'Fixed bug XXXXX: blah blah'
'User Story XXXXX: Blah blah' etc etc

Also this isn't super clear as to the implications of this - i.e., Why has it been removed? in favour of what? We share these with Service Support to help them understand changes.

Comment (dneed-nimble) [reply release-notes.md:2]: I only think the context is extra important on this one as it is a column change that will get queried by the Service support team.

**Review by dneed-nimble — APPROVED** [review]

**Review by dneed-nimble — APPROVED** [review]

**Pull request merged**